protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -60,10 +60,21 @@ Input data:
 
 Steps:
 
+1. From the [price monitoring module](./0032-PRIM-price_monitoring.md#view-from-quant-library-side) get the tightest `min_price_mon_price` and `max_price_mon_price`.
+From the market parameter `market.liquidity.volumeRange` which is a percentage price move (e.g. `0.05 = 5%` and from `mid_price` calculate 
+```
+min_lp_vol_price = max(min_price_mon_price, (1.0 - market.liquidity.volumeRange) x mid_price)
+``` 
+and 
+```
+max_lp_vol_price = min(max_price_mon_price, (1.0 + market.liquidity.volumeRange) x mid_price).
+```
+
 1. Calculate `liquidity_obligation`, as per calculation in the [market making mechanics spec](./0044-LIME-lp_mechanics.md).
 
-1. Subtract from the value obtained from step-1 the amount of the `liquidity_obligation` that is being fulfilled by any persistent orders the liquidity provider has on the book at this point in time according to the probability weighted liquidity measure (see [spec](../protocol/0034-PROB-prob_weighted_liquidity_measure.ipynb)). If you end up with 0 or a negative number, stop, you are done. 
-Note that the book `mid-price` must be used when calculating the probability weighted liquidity measure. 
+1. Subtract from the value obtained from step-1 the amount of the `liquidity_obligation` that is being fulfilled by any persistent orders the liquidity provider has on the book at this point in time that are between and including `min_lp_vol_price` and `max_lp_vol_price`. 
```

Comment:
```suggestion
1. The `liquidity_obligation` calculated as per [LP mechanics](0044-LIME-lp_mechanics.md). Some of it may be fulfilled by persistent orders the liquidity provider has on the book at this point in time that are between and including `min_lp_vol_price` and `max_lp_vol_price`. 
```